snow/engine/common/config.go

```diff
@@ -31,13 +29,6 @@ type Config struct {
 	// Max number of times to retry bootstrap before warning the node operator
 	RetryBootstrapWarnFrequency int
 
-	// Max time to spend fetching a container and its ancestors when responding
-	// to a GetAncestors
-	MaxTimeGetAncestors time.Duration
-
-	// Max number of containers in an ancestors message sent by this node.
-	AncestorsMaxContainersSent int
-
```

Comment:
I'm going to try to completely remove this struct... So this is a good start.

Reply:
will there be code duplication across avalanche and snowman configs?

Reply:
So the end goal here is going to be for the `node` package to define it's own config (since it's the one who is responsible for parsing configs it should also define them) and individually pass all required fields to downstream packages via constructor methods right?

Reply:
tbh that's probably a bit more grandiose than I'm really targeting right now. I'm trying to remove the usage of `common.Config` by the avalanche engine (which will in turn let it be inlined by the snowman engine)